inc/ubm_common.h

```diff
@@ -89,6 +89,8 @@
 //BP unique ID

```

Comment:
Commit message just mentions "support Volcano U.3 BP auto configuration.". However the comments have been modified in the source file. Please mention that in the commit message that comments have been updated.